index.php

```diff
@@ -7,6 +7,37 @@
 // Creates a variable which points to the correct database and gives username and password
 $db = new PDO('mysql:host=db;dbname=dinosaurs;', 'root', 'password');
 
+/**
+ * Checks if the form has been submitted and if so returns the dinos from the submitted search. If not, returns all dinos
+ *
+ * @param PDO $db database
+ * @return string of dinos to be displayed
+ */
+function checkIfSearched(PDO $db): string {
+    if (isset($_GET['submit'])) {
+        return Museum::displayAllDinos(DinosaurHydrator::getSearchedDinos($db, $_GET['search'], $_GET['filter']));
+    } else {
+        return Museum::displayAllDinos(DinosaurHydrator::getAllDinos($db));
+    }
+}
+
+/**
+ * Checks if the form has been submitted and if so returns the value the user searched for, otherwise returns search
+ *
+ * @return string
+ */
+function searchedValue(): string {
+    if (isset($_GET['submit'])) {
+        return $_GET['search'];
```

Comment:
check if GET['search'] exists

Reply:
Asked Ash and should always exist if submit exists as when the form is submitted if values haven't been filled in then search will be an empty string :)